docs/guide/hinge-2-joint-with-backlash.md

```diff
@@ -0,0 +1,66 @@
+
+### Hinge2JointWithBacklash Model
+
+```
+Hinge2JointWithBacklash [
+  SFNode       jointParameters     HingeJointParameters {}
+  SFNode       jointParameters2    JointParameters {}
+  SFFloat      backlash            0.01        # [0, inf)
+  SFFloat      backlash2           0.01        # [0, inf)
+  SFFloat      gearMass            0.01        # (0, inf)
+  MFNode       device              [ ]         # {RotationalMotor, PositionSensor, Brake}
+  MFNode       device2             [ ]         # {RotationalMotor, PositionSensor, Brake}
+  MFNode       outputSensor        [ ]         # {PositionSensor}
+  MFNode       outputSensor2       [ ]         # {PositionSensor}
+  MFNode       startPoint          NULL        # {Group, Transform, or Shape}
+  SFNode       endPoint            NULL        # {Solid, SolidReference, or Slot}
+]
+```
+
+The backlash effect is modeled by concatenating two [HingeJoint](../reference/hingejoint.md), one perpendicular to the other, to a classic [Hinge2Joint](../reference/hinge2joint.md).
+The axes of the [Hinge2Joint](../reference/hinge2joint.md) can freely turn whereas the range of movement of the two [HingeJoint](../reference/hingejoint.md) is limited by setting the values of `minStop` and `maxStop` according to the desired `backlash` in each direction.
+By doing so, the `endPoint` will begin to move only when either limit is reached.
+
+Given the nature of the implementation, this [PROTO](../reference/proto.md) won't achieve the desired effect if used in kinematic mode and therefore it requires the `endPoint` to have physics enabled.
+
+### Field Summary
+
+- `backlash`: this field specifies the amount of clearance in the transmission associated to `axis` of the [Hinge2Joint](../reference/hinge2joint.md). The backlash is defined as the maximum distance that the driving gear tooth can travel without any actual contact or transfer of power to the driven gear.
+If set to zero, the limiter in this direction is removed and therefore the corresponding fields `gearMass` and `outputSensor` are ignored.
+If both `backlash` and `backlash2` are set to zero, `startPoint` is also ignored and the [PROTO](../reference/proto.md) reverts to a classic [Hinge2Joint](../reference/hinge2joint.md).
+
+- `backlash2`: this field specifies the amount of clearance in the transmission associated to `axis2` of the [Hinge2Joint](../reference/hinge2joint.md).
+If set to zero, the limiter in this direction is removed and therefore the corresponding fields `gearMass2` and `outputSensor2` are ignored.
+If both `backlash` and `backlash2` are set to zero, `startPoint` is also ignored and the [PROTO](../reference/proto.md) reverts to a classic [Hinge2Joint](../reference/hinge2joint.md).
+
+- `gearMass`: this field specifies the mass of the transmission. If both `backlash` and `backlash2` are defined, then this mass is split among both axes, otherwise it's entirely assigned to the axis with non-zero backlash.
```

Comment:
Why not having `gearMass` and `gearMass2`? That would be more capable and easier to understand.

Reply:
You're right, it's more intuitive anyway.